draft-ietf-cdni-request-routing-extensions.xml

```diff
@@ -382,7 +405,124 @@
       </section> 
     </section>
 
-      
+
+    <section anchor="cdni-metadata-footprint-types" title="CDNI Metadata Footprint Types">
+      <t>The CDNI Footprint Object is defined in the CDNI Metadata interface document, see section 4.2.2.2 of
+          <xref target="RFC8006"/>. The object is composed of a footprint-type, referring to some of the simple data-types
+          defined in section 4.3 of <xref target="RFC8006"/>, as well as an array of properties from the specified type.
+          Additionally, in order to facilitate future specification of other optional "footprint-types",
+          a clear process for specifying optional footprint types in an IANA registry is also specified
+          (see section 7.2 of <xref target="RFC8006"/>).
+      </t>
+      <t>The FCI Capability Advertisement Object also uses the CDNI Footprint object, as described in section 5 of
+          <xref target="RFC8008"/>. The object is built of "capability-type", "capability-value", and a "footprints"
+          array built of CDNI Footprint Objects, to specify the clients to which the capability applies.
+      </t>
+
+      <t>This section specify new data-types, extending both <xref target="RFC8006"/> and <xref target="RFC8008"/>.
+      </t>
+
+
+      <section anchor="cdni-metadata-ipv4v6cidr-data-type" title="CDNI Metadata IPv4v6CIDR Data Type">
```

Comment:
Here add the sentence I have suggested in the previous comment, providing you accept my comment ...

Reply:
Second part of the next paragraph can also be removed, but I feel comfortable with it.

This section specifies an extension to the CDNI Metadata footprint types defined in section 4.2.2.2 of <xref target="RFC8006"/>. The footprint type is based on a CDNI Metadata simple data type, also defined as an extension of 4.3 of <xref target="RFC8006"/>.